github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -1019,3 +1020,28 @@ postsubmits:
                 memory: "8Gi"
               limits:
                 memory: "8Gi"
+    - name: mirror-images-from-docker-hub
```

Comment:
Can you please rename the job to add the prefix `post-project-infra-`?

Reply:
Done